Arrowgene.Ddon.Server/GameLogicSetting.cs

```diff
@@ -53,6 +76,26 @@ public GameLogicSetting()
             RookiesRingBonus = 1.0;
             NaiveLobbyContextHandling = true;
             CraftConsumableProductionTimesMax = 10;
+
+            AdjustPartyEnemyExp = true;
```

Comment:
Do we want these enabled or disabled by default?

Reply:
I'd say go for it, it's easily toggleable back to how it was before